src/index.ts

```diff
@@ -1,14 +1,14 @@
-import {
-  Web3,
-  HttpClient,
-  ProofPoint,
-  ProofPointIssueResult,
-  ProofPointRegistry,
-  ProofPointValidateResult,
-  ProofPointStatus,
-  ProofPointEventType,
-  ProofPointEvent,
-} from "./proofPoints";
+import Web3 from "web3";
+import { HttpClient } from "./httpClient";
+import { ProofPoint } from "./proofPoint";
+import { ProofPointIssueResult } from "./proofPointIssueResult";
```

Comment:
They could. Using namespaces triggers this eslint rule: 

https://github.com/typescript-eslint/typescript-eslint/blob/v2.34.0/packages/eslint-plugin/docs/rules/no-namespace.md

Do you think we should switch the rule off and use namespaces? What is the advantage?

Reply:
Could you use the `declare` method that page talks about? 

I suppose it just feels like there is a natural namespace there, because there are so many classes that start with `ProofPoint`. Its not a big deal but could be cleaner?